exercises/D-logical-operators/exercise2.js

```diff
@@ -5,7 +5,38 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(num) {
+  if (num < 0) {
+    return true;
+  } 
+    return false;
+  
+}
```

Comment:
This is good, but you are able to write this in one line:
```
function isNegative(num) {
 return num < 0;
}
```

This works because `return num < 0;` will return either `true` or `false`